Java/ds/NodetoRootPath.java

```diff
@@ -0,0 +1,77 @@
+import java.util.ArrayList;
```

Comment:
Add a beginner friendly pitch

Reply:
Can you pls elaborate this thing , as I can't understand this

Reply:
@ankitaggarwal23  Tell me this thing , I asked this  doubt 2 days ago 
Tell me how many of you answered this

Reply:
@ankitaggarwal23 Sir , can you tell this  why Harsh means by Beginner Friendly Pitch  for this line